server/controllers/documentsControllers.js

```diff
@@ -103,12 +103,21 @@ export default {
         order: '"createdAt" DESC'
       })
         .then((documents) => {
-          const settings = limit && offset ? {
-            totalCount: documents.count,
-            pages: Math.ceil(documents.count / limit),
-            currentPage: Math.floor(offset / limit) + 1,
-            pageSize: documents.rows.length
-          } : null;
+          // const settings = limit && offset ? {
```

Comment:
Thanks for that. It appears you are probably looking at the wrong branch. https://github.com/andela-aalabi/DMS-papyrus/blob/feedback/server/controllers/documentsControllers.js